pkg/build/builder/common.go

```diff
@@ -92,7 +92,7 @@ func buildInfo(build *buildapi.Build, sourceInfo *git.SourceInfo) []KeyValue {
 // that the built image can be referred to unambiguously even in the face of
 // concurrent builds with the same name in the same namespace.
 func randomBuildTag(namespace, name string) string {
-	repo := fmt.Sprintf("%s/%s", namespace, name)
+	repo := fmt.Sprintf("docker.io/%s/%s", namespace, name)
```

Comment:
Unfortunately this is only canonical on the Red Hat Docker version...

Reply:
the value doesn't actually matter, this tag isn't used for anything except to have a local temporary tag that wont conflict w/ anything and that we can use to refer to the image when running it in a post-commit hook.  The docker.io was added because apparently w/ a docker vendor update, the docker client stopped liking image names that don't have some repository value specified.

Reply:
correct, all repositories must be canonical from now.